### PR TITLE
Jesse dev

### PR DESCRIPTION
**Pull request by jnayak1**
Added ability for admins to accept/decline registration from a project registration page. Fixed bug that caused it to crash when no user was signed in.

### CODE REVIEW

File: website/templates/project/project_header.mako [hunk 1]
```diff
@@ -109,9 +110,56 @@
...
+                        </div>
+                        <%def name="javascript_bottom()">
+                            console.log('hello')
+                            ${parent.javascript_bottom()}
+                            % for script in addon_page_js or []:
+                                <script type="text/javascript" src="${script}"></script>
+                            % endfor
+                        </%def>
+                        <%doc>
+                            <script type="text/javascript">
```
Comment (sloria) [inline website/templates/project/project_header.mako]: Remove any unused code

File: website/templates/project/project_header.mako [hunk 2]
```diff
@@ -109,9 +110,56 @@
...
+                                <a href="${node['approval_link']}"></a>
+                                Approve Registration
+                            </button>
+                            <button type="button" id="registrationCancelButton" class="btn btn-danger">
+                                <a href="${node['approval_link']}"></a>
+                                Cancel Registration
+                            </button>
+                        </div>
+                        <%def name="javascript_bottom()">
+                            console.log('hello')
```
Comment (sloria) [inline website/templates/project/project_header.mako]: Remove debugging code

File: website/templates/project/project_header.mako [hunk 3]
```diff
@@ -109,9 +110,56 @@
...
+                            <button type="button" class="btn btn-success">
+                                <a href="${node['approval_link']}"></a>
+                                Approve Registration
+                            </button>
+                            <button type="button" id="registrationCancelButton" class="btn btn-danger">
+                                <a href="${node['approval_link']}"></a>
+                                Cancel Registration
+                            </button>
+                        </div>
+                        <%def name="javascript_bottom()">
```
Comment (sloria) [inline website/templates/project/project_header.mako]: Is this block necessary?

**Comment by Ghalko** [comment]
@GaryKriebel This should be merged soon.

**Pull request merged**